src/App.test.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme';
 import App from './App';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+ describe('App', ()=>{
+  it('Should render without crashing', function(){
+    const app = shallow(<App/>)
+    expect(app.length).toBe(1)
+
+   });
 });
+
+describe('Addition', () => {
```

Comment:
Let's remove this test. The first one can serve as an example. We may merge this and later forget to delete it, which brings more issues.

Reply:
Noted working on it right now